src/locale/locales/pt-BR/messages.po

```diff
@@ -1626,7 +1626,7 @@ msgstr "Copiar texto do post"
 
 #: src/components/StarterPack/QrCodeDialog.tsx:171
 msgid "Copy QR code"
-msgstr ""
+msgstr "Copiar código QR "
```

Comment:
For us its okay read "Copiar QR code"

Reply:
Thank you!! Feel free to leave more notes or anything too <3

Reply:
```suggestion
msgstr "Copiar código QR "
```